.github/workflows/build-dev-release.yml

```diff
@@ -0,0 +1,15 @@
+name: Build and publish dev release Docker image to Github Container Registry ghcr.io
```

Comment:
note: the `workflow_dispatch` does not work until the workflow is merged into the default branch (`master` for this repo). We either have to merge dev-version-2 to the master or make dev-version-2 default branch